server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
```

Comment:
The constant variables PORT and HOSTNAME should be named in uppercase letters as it is a common convention for constants.